packages/cardano-services/src/Program/loadHttpServer.ts

```diff
@@ -11,19 +11,23 @@ import { DbSyncUtxoProvider, UtxoHttpService } from '../Utxo';
 import { DnsResolver, createDnsResolver } from './utils';
 import { HttpServer, HttpServerConfig, HttpService } from '../Http';
 import { InMemoryCache } from '../InMemoryCache';
+import { Logger } from 'ts-log';
 import { MissingProgramOption, UnknownServiceName } from './errors';
 import { ServiceNames } from './ServiceNames';
+import { SrvRecord } from 'dns';
 import { TxSubmitHttpService } from '../TxSubmit';
 import { createDbSyncMetadataService } from '../Metadata';
+import { createLogger } from 'bunyan';
 import { getOgmiosCardanoNode, getOgmiosTxSubmitProvider, getPool, getRabbitMqTxSubmitProvider } from './services';
-import Logger, { createLogger } from 'bunyan';
 import memoize from 'lodash/memoize';
 import pg from 'pg';
 
 export interface HttpServerOptions extends CommonProgramOptions {
   serviceNames?: ServiceNames[];
   enableMetrics?: boolean;
   cardanoNodeConfigPath?: string;
+  dnsResolver?: (serviceName: string) => Promise<SrvRecord>;
+  logger?: Logger;
```

Comment:
I would put these in a separate object (2nd arg to loadHttpServer). `HttpServerDependencies`?

Reply:
I spent at least half an hour this morning thinking and writing about standardization of our interfaces and I lost this...
Thank you for the suggestion!